modules/context/repo.go

```diff
@@ -577,7 +577,8 @@ func RepoAssignment(ctx *Context) (cancel context.CancelFunc) {
 	ctx.Data["BranchName"] = ctx.Repo.BranchName
 
 	// People who have push access or have forked repository can propose a new pull request.
-	canPush := ctx.Repo.CanWrite(unit_model.TypeCode) || (ctx.IsSigned && ctx.User.HasForkedRepo(ctx.Repo.Repository.ID))
+	canPush := ctx.Repo.CanWrite(unit_model.TypeCode) || (ctx.IsSigned &&
+		models.HasForkedRepo(ctx.User.ID, ctx.Repo.Repository.ID))
```

Comment:
Unusual place for a newline to me...
Maybe move the ctx.IsSigned to the next line as well?